SPECS/openvswitch/openvswitch.spec

```diff
@@ -278,6 +279,9 @@ make -k check |& tee %{_specdir}/%{name}-check-log || %{nocheck}
 %{_mandir}/man8/ovn-trace.8.gz
 
 %changelog
+* Mon Apr 19 2021 Nicolas Ontiveros <niontive@microsoft.com> - 2.12.3-2
+- Don't include static libraries in openvswitch package
```

Comment:
This is a breaking change. Please make sure that we either have no derivatives making used of this package (or at least the static libs) or help them switch to using the `*-static` subpackage instead.